README.md

```diff
@@ -29,15 +29,14 @@ You should (eventually) see output similar to the following:
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
 ## Running Tests
-These can be run in a docker container or locally
-you will need to run `poetry install` in order to run the tests locally, or build the docker container once with `docker build --target test --tag todoapp-test .` to run tests in docker
-To run all tests in docker: `docker run --env-file ./.env todoapp-test`
+These can be run in a docker container or locally. You will need to run `poetry install` in order to run the tests locally, or build the docker container once with `docker build --target test --tag todoapp-test .` to run tests in docker. 
+To run all tests in docker: `docker run --env-file ./.env todoapp-test`. 
 To run all the tests locally: `poetry run pytest`
 
 ### Unit tests
-to run them (and the integration tests) in the docker container: `docker run todoapp-test src/tests`
+To run them (and the integration tests) in the docker container: `docker run todoapp-test src/tests`
 
-to run them locally:
+To run them locally: 
 navigate to the the `src/tests` directory
 run `poetry run pytest` to run all the unit and integration tests
 run `poetry run pytest <filename>` to run the unit tests in that file
```

Comment:
Your README should document getting setup with OAuth, i.e. creating a GitHub oauth app with the correct redirect URLs and setting the client id and secret in .env.